frontend/app/views/layout.server.view.html

```diff
@@ -31,10 +31,15 @@
 </head>
 
 <body class="ng-cloak">
-	<header data-ng-include="'/modules/core/views/header.client.view.html'" class="navbar navbar-fixed-top"></header>
+
+  <header data-ng-include="'/modules/core/views/header.client.view.html'"
+  class="navbar navbar-fixed-top navbar-inverse"></header>
+
 	<section class="content">
 		<section class="container">
-			{% block content %}{% endblock %}
+			{% block content %}
```

Comment:
Testing, should have deleted

Jen

jtrudell@gmail.com 
312-218-6381 (cell)
jtrudell78 (Skype)

> On Oct 12, 2015, at 6:54 AM, Brian Turnbull <notifications@github.com> wrote:
> 
> In frontend/app/views/layout.server.view.html:
> 
> >  	<section class="content">
> >  		<section class="container">
> > -			{% block content %}{% endblock %}
> > +			{% block content %}
> Is this necessary or were you just testing?
> 
> —
> Reply to this email directly or view it on GitHub.
>